windows/src/engine/keyman32/k32_lowlevelkeyboardhook.cpp

```diff
@@ -194,17 +194,24 @@ LRESULT _kmnLowLevelKeyboardProc(
 
       HWND hwnd = gui.hwndFocus ? gui.hwndFocus : gui.hwndActive;
       if (!IsConsoleWindow(hwnd)) {
-        PostThreadMessage(GetWindowThreadProcessId(hwnd, NULL), wm_keyman_keyevent, hs->vkCode, LLKHFFlagstoWMKeymanKeyEventFlags(hs));
+#ifdef USE_KEYEVENTSENDERTHREAD
+        PostMessage(hwndKeyEventSender, wm_keyman_keyevent, hs->vkCode, LLKHFFlagstoWMKeymanKeyEventFlags(hs));
+#else
+        PostThreadMessage(GetWindowThreadProcessId(GetForegroundWindow(), NULL), wm_keyman_keyevent, hs->vkCode, LLKHFFlagstoWMKeymanKeyEventFlags(hs));
+#endif
         return 1;
       }
       else {
-        SendDebugMessageFormat(0, sdmGlobal, 0, "LowLevelHook: console window, not serializing");
+        //SendDebugMessageFormat(0, sdmGlobal, 0, "LowLevelHook: console window, not serializing"); // noisy 
       }
     }
     else {
       SendDebugMessageFormat(0, sdmGlobal, 0, "LowLevelHook: Failed to get Gui thread info with error %d", GetLastError());
     }
+    
+    return 1;
```

Comment:
TODO @mcdurdin: I need to remove this; it's an error condition so it should fall back to default behaviour and not trap keystrokes.